controller_manager/test/test_controller_manager.cpp

```diff
@@ -33,6 +33,34 @@ class TestControllerManagerWithStrictness
 {
 };
 
+class TestControllerManagerRobotDescription
+: public ControllerManagerFixture<controller_manager::ControllerManager>
+{
+};
+
+TEST_F(TestControllerManagerRobotDescription, controller_robot_description_update)
+{
+  auto test_controller = std::make_shared<test_controller::TestController>();
+  auto test_controller2 = std::make_shared<test_controller::TestController>();
+  constexpr char TEST_CONTROLLER2_NAME[] = "test_controller2_name";
```

Comment:
Could this be auto without []? And maybe outside of this function

Reply:
Sure. Done!